test/common/common/logger_fuzz_test.cc

```diff
@@ -0,0 +1,14 @@
+#include "common/common/logger.h"
+
+#include "test/fuzz/fuzz_runner.h"
+
+namespace Envoy {
+namespace Fuzz {
+
+DEFINE_FUZZER(const uint8_t* buf, size_t len) {
+  Logger::DelegatingLogSink::escapeLogLine(
```

Comment:
I think we would be better off having a full logging fuzz test. I appreciate folks adding fuzz tests as they go, but if we could get more coverage, i.e. by instantiating a logging context with different config options (escape, no-escape, randomized format strings) as well as having fuzzed inputs, we make better use of the cycles on ClusterFuzz. CC @asraa

Reply:
I agree. Something at the `DelegatingLogSink::log` level would even be great.
Another thing you could explore is the equivalence of the espaced log and the unescaped log (modulo trailing whitespace).

Reply:
I'm willing to add these tests, but could this be done in a separate PR? I'll have to spend some time reading through the rest of the logger code to understand how to test it. Also this PR is becoming rather large, we were hoping to get this in soon to enable Stackdriver Integration in our codebase. 

I can do it in this PR if required, but it may take a couple of days to get around to it.

Reply:
Thank you! Yeah, I would be happy to see the fuzz test in a later PR. In that case could you add a `no_fuzz` tag to the build target with a TODO to add more coverage to the fuzz test? We filters these out on OSS-Fuzz. See https://github.com/envoyproxy/envoy/blob/160cf76ad3261094f7c7073c60a5b00a28d12787/test/common/protobuf/BUILD#L46

Reply:
Done. Created #8893